test/FunctionalTests/Microsoft.OData.Core.Tests/IntegrationTests/Evaluation/AutoComputePayloadMetadataInJsonIntegrationTests.cs

```diff
@@ -1366,8 +1364,8 @@ private static IEdmModel GetModelWithAbstractType(out EdmEntitySet entitySet)
         public void WritingContainedNavigationPropertyWithoutProvideKeyPropertyOnParentResourceWorks(string metadataLevel)
         {
             MemoryStream outputStream = new MemoryStream();
-            var container = ContainerBuilderHelper.BuildContainer(null);
-            IODataResponseMessage message = new InMemoryMessage() { Stream = outputStream, Container = container };
+            var container = ServiceProviderHelper.BuildServiceProvider(null);

```

Comment:
I know this is nitpicky, but changing the variable name to `serviceProvider` here, and other places in this file, would potentially reduce confusion.